app/static/drugset.js

```diff
@@ -1,108 +1,109 @@
-        }
-    }
-    return dataArray;
-}
-
-function ds_drawTable(reviewed) {
+function ds_drawTable(url, reviewed) {
     let columns = [
-        {title: "Description"},
-        {title: "Drugs"},
-        {title: "Hidden drugs"}
+        {title: "Description", data: 'descrShort'},
+        {title: "Drugs", data: 'drugs', orderable: false},
     ];
     if (reviewed === 0) {
-        columns.push({title: "Review"});
+        columns.push({title: "Review", data: 'id', orderable: false});
     }
 
-    $.get("drugsets", {reviewed: reviewed}, function (data) {
-        let table = $('#drugset_table').DataTable({
-            width: '100%',
-            data: ds_DTblify(JSON.parse(data), reviewed),
-            responsive: true,
-            columns: columns,
-            dom: 'B<"small"f>rt<"small row"ip>',
-            buttons: [],
-            columnDefs: [
-                {
-                    targets: 0,
-                    width: '70%'
-                },
-                {
-                    targets: 1,
-                    width: '30%'
-                },
-                {
-                    targets: 2,
-                    visible: false
+    let columnDefs = [
+        {
+            targets: 0,
+            width: '70%',
+            render: function (data, type, row) {
+                const showContacts = row['showContacts'];
+                const contacts = showContacts ? `<p><b>Author:</b> ${row['authorName']}<\p><p><b>Affiliation:</b> ${row['authorAffiliation']}<\p><p><b>E-mail:</b> ${row['authorEmail']}<\p>` :
+                    reviewed ? '<i class="far fa-eye-slash"></i> Author preferred not to share contact details' : `<p style="color: red"><i class="far fa-eye-slash"></i> As author preferred not to share contact details, following will not be displayed:</p><p style="color: red"><b>Author:</b> ${row['authorName']}<\p><p style="color: red"><b>Affiliation:</b> ${row['authorAffiliation']}<\p><p style="color: red"><b>E-mail:</b> ${row['authorEmail']}<\p>`;
+                let source;
+                if (row['source'] === null) {
+                    source = ''
+                } else if ((row['source'].includes('http://')) || (row['source'].includes('https://'))) {
+                    // let a = document.createElement('a');
+                    // a.href = row['source'];
+                    source = `<p><b>Source: </b><a class="wrapped" href="${row['source']}"  target="_blank">${row['source']}</a></p>`
+                } else {
+                    source = `<p><b>Source: </b> ${row['source']}</p>`;
                 }
-            ],
-            language: {
-                search: "Search in description or drugs:",
-                searchPlaceholder: "e.g. Remdesivir",
+                const date = `<p><b>Date added:</b> ${row['date'].split(' ')[0]}</p>`;
```

Comment:
And here date is `Fri, 10 Apr 2020 16:48:27 GMT`
Which is also not accurate timewise